common/dynamicconfig/constants.go

```diff
@@ -3871,6 +3879,11 @@ var BoolKeys = map[BoolKey]DynamicBool{
 		Description:  "Enable shardId metrics in persistence client",
 		DefaultValue: true,
 	},
+	EnableTTL: DynamicBool{
```

Comment:
We need a better name. e.g. EnableExecutionTTL indicating that it's for execution table